src/status_im/constants.cljs

```diff
@@ -42,38 +43,43 @@
     (= testnet-id rinkeby-id) "Rinkeby"
     :else "Unknown"))
 
+(def mainnet-networks
+  {"mainnet"     {:id     "mainnet",
+                  :name   "Mainnet",
+                  :config {:NetworkId mainnet-id
+                           :DataDir   "/ethereum/mainnet"}}
+   "mainnet_rpc" {:id     "mainnet_rpc",
+                  :name   "Mainnet with upstream RPC",
+                  :config {:NetworkId      mainnet-id
+                           :DataDir        "/ethereum/mainnet_rpc"
+                           :UpstreamConfig {:Enabled true
+                                            :URL     "https://mainnet.infura.io/z6GCTmjdP3FETEJmMBI4 "}}}})
```

Comment:
Extra space at the end?

Reply:
fixed, though it worked even with this extra space